products/idn/api/patch-requests.md

```diff
@@ -0,0 +1,500 @@
+        "since": "2021-01-22T14:48:58.072Z",
+        "status": "SOURCE_STATE_ERROR_ACCOUNT_FILE_IMPORT"
+    },
+```
+
+</details>
+
+You can send a PATCH request to make changes to paths within the "connectorAttributes" path itself. This example request removes the "filterString" path and its value. 
+
+```text
+PATCH https://{tenant}.api.identitynow.com/v3/sources/:id
+```
+
+```json
+[
+  {
+    "op": "remove",
+    "path": "/connectorAttributes/filterString",
+    "value": "!( id.contains( \"m\" ) ) || !( id.contains( \"d\" ) )"
+  }
+]
+```
+
+This request specifies the top-level path, "connectorAttributes", before following it with the path where the value is going to be removed from, "filterString". 
+
+## Specify a value 
+
+For some of the operations, once you have specified the operation you want to apply and the path you want to apply it to, you need to specify the value you want to send with the request. The value you specify must fit the path's data type. 
+
+You can specify a single simple value for an operation. In this example from earlier, the PATCH request replaces the source's description: 
+
+```text
+PATCH https://{tenant}.api.identitynow.com/v3/sources/:id
+```
+
+```json
+[
+  {
+    "op": "replace",
+    "path": "/description",
+    "value": "new description"
+  }
+]
+```
+
+You can also specify multiple values for an operation to be applied to. For example, this PATCH request replaces the source's current features with a number of new ones, all in the same path: 
+
+```text
+PATCH https://{tenant}.api.identitynow.com/v3/sources/:id
+```
+
+```json
+[
+  {
+    "op": "replace",
+    "path": "/features",
+    "value": [
+      "PASSWORD",
+      "PROVISIONING",
+      "ENABLE",
+      "AUTHENTICATE"
+    ]
+  }
+]
+```
+
+A value can also be an object that contains other values within it. For example, this PATCH request adds a new "location" attribute to the end of the source schema's array of attributes: 
+
+```text
+PATCH https://{tenant}.api.identitynow.com//v3/sources/:sourceId/schemas/:schemaId
+```
+
+```json
+[
+  {
+    "op": "add",
+    "path": "/attributes/-",
+    "value": {
+      "name": "location",
+      "type": "STRING",
+      "schema": null,
+      "description": "Employee location",
+      "isMulti": false,
+      "isEntitlement": false,
+      "isGroup": false
+    }
+  }
+]
+```
+
+This request uses the [PATCH Source Schema](https://developer.sailpoint.com/idn/api/v3/update-source-schema) endpoint to add a new attribute, along with its details, to the end of the array of a source's schema's attributes. 
+
+This example uses the "-" after the path to indicate that the value will be added to the end of the array. You can specify the position in the array "/" following the path. For example, if there were a list of 3 attributes, you could specify "/attributes/2" and the request would add the new "location" attribute to the second position in the array. 
+
+## Specify a from 
```

Comment:
Ah yeah, I'll make this info more explicit about which require values and which don't. I think I may leave it as a heading 2 though, because I think it's an alternative to value, but I also think of it as a same-level input.